README.rst

```diff
@@ -30,27 +30,27 @@ You're all set! Retrieve the list of available indices by running in your termin
 
 ::
 
-    run-func search getIndices
+    run-func index.js getExistingIndices
 
 
 How to use
 ----------
 
-1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to inject data::
+1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to index data:
 
-    run-func index injectData
+    run-func index.js indexData
 
 Wait till the data is indexed. Can take 15-20 seconds.
 
 2. Check `search.js` and `aggregate.js` for examples you can run. Every method contains a terminal command sample that you can try.
 
 ::
 
-    run-func search match title "garlic onion cake"
+    run-func search.js match title "garlic onion cake"
```

Comment:
None of these functions Worked on My Computer ™️  without adding the .js at the end. It would be useful to have someone else try if we have time before the workshop.